docs/extensions-plugins/cli/commands.mdx

```diff
@@ -72,13 +72,23 @@ Lists all accessible AI models.
 pieces list models
 ```
 
-### `open`
-Opens an asset from a list or search. If only "open" is used, it opens the most recent asset.
+#### open (deprecated)
```

Comment:
Also, this should be an h3 (###) not an h4.

And can we make sure to keep the back ticks around `open`?

Reply:
addressed in commit [3f067e5](https://github.com/pieces-app/documentation/pull/538/commits/3f067e5a05d2265c3e7e72e21d93b50bafb7e6bd)